cmd/bee-split/main.go

```
@@ -28,125 +23,22 @@ var (
-	for _, ch := range chs {
-		chunkPath := filepath.Join(f.path, ch.Address().String())
-		err := ioutil.WriteFile(chunkPath, ch.Data(), 0o666)
-		if err != nil {
-			return nil, err
-		}
-	}
-	return nil, nil
-}
-
-// apiStore provies a storage.Putter that adds chunks to swarm through the HTTP chunk API.
-type apiStore struct {
-	baseUrl string
-}
-
-// newApiStore creates a new apiStor
-func newApiStore(host string, port int, ssl bool) storage.Putter {
-	scheme := "http"
-	if ssl {
-		scheme += "s"
-	}
-	u := &url.URL{
-		Host:   fmt.Sprintf("%s:%d", host, port),
-		Scheme: scheme,
-		Path:   "bzz-chunk",
-	}
-	return &apiStore{
-		baseUrl: u.String(),
-	}
-}
-
-// Put implements storage.Putter
-func (a *apiStore) Put(ctx context.Context, mode storage.ModePut, chs ...swarm.Chunk) (exist []bool, err error) {
-	c := http.DefaultClient
-	for _, ch := range chs {
-		addr := ch.Address().String()
-		buf := bytes.NewReader(ch.Data())
-		url := strings.Join([]string{a.baseUrl, addr}, "/")
-		res, err := c.Post(url, "application/octet-stream", buf)
-		if err != nil {
-			return nil, err
-		}
-		if res.StatusCode != http.StatusOK {
-			return nil, fmt.Errorf("upload failed: %v", res.Status)
-		}
-	}
-	return nil, nil
-}
-
-// limitReadCloser wraps the input to the application to limit the input to the given count flag.
-type limitReadCloser struct {
-	io.Reader
-	closeFunc func() error
-}
-
-// newLimitReadCloser creates a new limitReadCloser.
-func newLimitReadCloser(r io.Reader, closeFunc func() error, c int64) io.ReadCloser {
-	return &limitReadCloser{
-		Reader:    io.LimitReader(r, c),
-		closeFunc: closeFunc,
-	}
-}
-
-// Close implements io.Closer
-func (l *limitReadCloser) Close() error {
-	return l.closeFunc()
-}
-
 // Split is the underlying procedure for the CLI command
 func Split(cmd *cobra.Command, args []string) (err error) {
-	var infile io.ReadCloser
+	logger, err = cmdfile.SetLogger(cmd, verbosity)
+	if err != nil {
+		os.Exit(1)
```

Also here, we can return the error.